hack/update-generated-clientsets.sh

```diff
@@ -28,7 +28,7 @@ function generate_clientset_for() {
   grouppkg=$(realpath --canonicalize-missing --relative-to=$(pwd) ${package}/..)
   client-gen --clientset-path="${grouppkg}/generated" \
              --input-base="${package}"                            \
-             --output-base="../../.."                           \
+             --output-base="${GOPATH}/src"                            \
```

Comment:
There is no occurence of `$OSROOT/src` or `${OSROOT}/src` in the whole source code, while there are multiple for `$GOPATH`.

Specifically, `hack/lib/build/binaries.sh` says `local go_pkg_dir="${GOPATH}/src/${OS_GO_PACKAGE}"`.